app/controllers/api/v1/users_controller.rb

```diff
@@ -4,7 +4,7 @@ def index
 		all_users = User.all
 		users = []
 		all_users.each do |user|
```

Comment:
Please check here if the current user is an admin. If so, they can see all users like you have, but if they are not you must filter out all the admins.